drivers/serial/Kconfig

```diff
@@ -13,6 +13,16 @@ config NRF_SW_LPUART
 
 if NRF_SW_LPUART
 
+config NRF_SW_LPUART_INIT_PRIORITY
+	int "Initialization priority"
+	default 45
+	help
+	  Initialization priority within POST_KERNEL level. It should be set to
+	  a value that ensures that device is initialized later than gpio and
+	  uart devices used by this device. Default is set to be between
```

Comment:
How about adding a build assertion that will check if this condition (and also the priority value is lower than `KERNEL_INIT_PRIORITY_DEVICE` which is used by `bt_uart_init()` in h4.c) is met?

Reply:
Abandoned the idea after offline discussion.

Reply:
We talked offline and decided to not add assert for now.